extensions/functions_list.yaml

```
@@ -0,0 +1,42 @@
+%YAML 1.2
+---
+urn: extension:io.substrait:functions_list
+scalar_functions:
+  - name: "map"
+    description: >-
+      Maps each element of an array using a lambda function.
+
+      Returns a new array where each element is the result of applying
+      the map function to the corresponding element in the input array.
```

```suggestion
      the mapper function to the corresponding element in the input array.
```

You call the function `mapper` below